internal/app/user.go

```diff
@@ -0,0 +1,124 @@
+package app
+
+import (
+	"database/sql"
+	"encoding/json"
+	ihttp "github.com/TheVitik/sport-together-go/internal/http"
+	"github.com/dgrijalva/jwt-go"
+	"net/http"
+	"time"
+)
+
+// secretKey is the secret key used to sign JWT tokens
+var secretKey = []byte("secret")
+
+// Login authenticates a user and returns a JWT token
+func Login(db *sql.DB) http.HandlerFunc {
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		var credentials struct {
+			Email    string `json:"email"`
+			Password string `json:"password"`
+		}
+		err := json.NewDecoder(r.Body).Decode(&credentials)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusBadRequest)
```

Comment:
It's better to use some validators. For example `github.com/asaskevich/govalidator`